### PR TITLE
Add Simple interactive UI

### PR DESCRIPTION
**Pull request by rebaz36**
# in This part of the project I have completed the following:

- Add main.rb file to demonstrate a simple interactive UI
- Add app.rb file that we can access the combination of the application's feature

### CODE REVIEW

File: main.rb [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    else
+      puts 'Invalid choice.'
+    end
+
+    puts
+  end
+end
+
+# Invoke the main method
+main
```
Comment (prolajumokeoni) [inline main.rb:70]: ### OPTIONAL

```suggestion
main()
```

- Always add the `()` after a method, this is the conventional way of writing methods

File: main.rb [hunk 2]
```diff
@@ -0,0 +1,70 @@
+require './app' # import the App class
+
+def main # rubocop:disable Metrics/AbcSize, Metrics/MethodLength, Metrics/CyclomaticComplexity
+  app = App.new
```
Comment (prolajumokeoni) [inline main.rb:4]: ### OPTIONAL

```suggestion
  app = App.new()
```

- Always add the `()` after a method, this is the conventional way of writing methods

**Review by prolajumokeoni — APPROVED** [review]
Hi @rebaz36 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- No linter errors ✅ 
- Git flow was used correctly 🚀 
- Good Readme💯 
- Good Pr title and description 💪🏿 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
_Check inline comments_

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: main.rb (hunk 1 above)
Comment (rebaz36) [reply main.rb:70]: hey @prolajumokeoni thanks for the suggestions, i actually did it like this at first but when I do Rubocop is giving me an error and that's why I changed it to the way it is. like the example below 👇🏼 

Screenshot 2022-12-21 at 23 03 25

**Pull request merged**